helm/cluster-api-core/Chart.yaml

```diff
@@ -5,11 +5,5 @@ home: "https://github.com/giantswarm/cluster-api-core-app"
 version: "[[ .Version ]]"
 appVersion: [[ .AppVersion ]]
 annotations:
-  application.giantswarm.io/owners: |
-    - catalog: default
-      provider: aws
-      team: firecracker
-    - catalog: default
-      provider: azure
-      team: celestial
+  application.giantswarm.io/team: "phoenix"
```

Comment:
For now switching to the team annotation. Once we have CAPO and CAPV we can go back to routing by provider.